integration-tests/features/component_analysis_v2.feature

```diff
@@ -0,0 +1,246 @@
+          And I should receive a valid JSON response
+
+   
+  Scenario: Check the component analysis REST V2 API endpoint for unknown component in NPM ecosystem
+    Given System is running
+      Given Three scale preview service is running
+  
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+    
+     When I start v2 component analyses npm/really_unknown_component/1.0.0 with user_key
+      Then I should get 202 status code
+      And I should receive a valid JSON response
+
+  
+  Scenario: Check the component analysis V2 REST API endpoint for unknown component in PyPi ecosystem
+    Given System is running
+     Given Three scale preview service is running
+      
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+
+     When I start v2 component analyses pypi/really_unknown_component/1.0.0 with user_key
+     Then I should get 202 status code
+      And I should receive a valid JSON response
+      
+    
+  Scenario: Check if the snyk login link works
+    Given System is running
+    Given Three scale preview service is running
+    
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+
+      When I start v2 component analyses npm/mongoose/4.2.4 with user_key
+      Then I should get 200 status code
+        And I should receive a valid JSON response
+      When I try to access snyk link i should get 200 status code
+  
+  Scenario: Check one npm package for private vulnerabilities
+    Given System is running
+    Given Three scale preview service is running
+    
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+
+      When I start v2 component analyses npm/lodash/4.17.4 with user_key
+      Then I should get 200 status code
+        And I should receive a valid JSON response
+      Then I should find one or more vulnerabilities in result with valid attributes
+      Then I should find a private vulnerability in v2 component analysis
+  
+  Scenario: Check one pypi package for private vulnerabilities
+    Given System is running
+    Given Three scale preview service is running
+      
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+
+      When I start v2 component analyses pypi/markdown2/2.2.0 with user_key
+      Then I should get 200 status code
+        And I should receive a valid JSON response
+      Then I should find one or more vulnerabilities in result with valid attributes
+      Then I should find a private vulnerability in v2 component analysis
+
+  Scenario: Check one npm package for no private vulnerabilities
+    Given System is running
+    Given Three scale preview service is running
+      
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+
+      When I start v2 component analyses npm/ejs/1.0.0 with user_key
```

Comment:
why only npm? what about the other 2 ecosystems?

Reply:
will add for other 2 ecosystems also